organization/voting.md

```diff
@@ -19,14 +26,47 @@ check conway's law: "Any organization that designs a system will produce a desig
 
 5 seems to not introduce too many established structures
 
+```{note}
+I follow the rationale. Now, should the collective start at 5? If not, what is the timeline to gradually get to this figure? Are processes defined elsewhere robust to size fluctuations?
+```
+
 ## Voting processes
 
 collective is composed of 5 people, each 1 vote. votes are cryptographically signed.<br>
 
-decision is taken with majority, and everyone needs to vote. Exceptionally, a decision can be taken by only 4 members if they all agree, i.e. all 4 vote for the same resolution.<br>
+decision is taken with majority, and everyone needs to vote. 
+
+```{note}
+[brainstorming]
+Given assumption above, how are negative votes handled? If vote is sync, we expect all members to be present at a given time and the vote decision YES/NO is taken. If vote is async, we expect all members to have produced a vote by a deadline.
+```
```

Comment:
decision/changes should be proposed as a shared document and votes should be added to it as cryptographic signatures in an asynchronous date. You are right to say that if the vote is asynchronous, it should probably have a deadline.
Format of the vote (ie: yes/no, or multiple choices, ...) is open and most likely specific to each question being voted upon.